src/xo-app/tasks/index.js

```diff
@@ -113,9 +107,9 @@ export default class Tasks extends Component {
     const { props, state } = this
     const {
       intl,
-      nTasks,
       pendingTasksByPool
     } = props
+    const nTasks = size(pendingTasksByPool)
```

Comment:
You can keep using the `.count()` method of the selector, you just don't need to pass a predicate anymore.